lib/src/main/java/com/what3words/components/vm/AutosuggestViewModel.kt

```diff
@@ -32,6 +32,10 @@ internal open class AutosuggestViewModel(
 
     var options: AutosuggestOptions = AutosuggestOptions()
 
+    fun display(suggestionWithCoordinates: SuggestionWithCoordinates) {
+        io(dispatchers) { _selectedSuggestion.emit(suggestionWithCoordinates) }
```

Comment:
any specific reason why we emit flows inside io dispatcher?

Reply:
Good find 👍